server/static/js/profile_page.js

```diff
@@ -99,11 +125,12 @@ function($, _, _s, _bootstrap, term, _course, friend, _util, user, _user_course,
     });
 
     // Add the parsed term and course info to the page for live preview
-    var profileTermsView = new term.ProfileTermsView({
+    this.profileTermsView = new term.ProfileTermsView({
       termCollection: termCollection,
       showAddTerm: window.pageData.ownProfile
     });
-    $('#profile-terms-placeholder').replaceWith(profileTermsView.render().el);
+    $('#profile-terms-placeholder').replaceWith(
+      this.profileTermsView.render().el);
```

Comment:
I should put something in the Contributing file about using React for new front-end things.. no more of this `render().el`